DerivedData/Build/Products/Debug-iphonesimulator/FluentIcons-iOS/FluentIcons.framework/_CodeSignature/CodeResources

```diff
@@ -0,0 +1,252 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Looks like some build output has snuck in here. Can we get rid of that and update the gitignore file?

Reply:
Sure, don't know how this file ended up here. :)

Reply:
#140